### DISCUSSION TITLE
Should SpeedAngleTriplet have a constructor without an angle, but a distance?

### DISCUSSION THREAD

**Discussion by GalexY727** [Patribots4738/Crescendo2024#138] [Ideas]
If our pivot's ideal angle is always a straight shot to the speaker, should we just put in the distance from the speaker into the triplet and have it run the calculation in ShooterCalc::calculatePivotAngle for its angle? I suppose only if the pivot's ideal angle is the straight shot to the speaker, it might not be in all cases.

**Comment by jacueblol** [comment]
I would keep this issue open until we are open for testing, but I like where your mind is.
